src/driver.ts

```diff
@@ -198,6 +224,17 @@ export abstract class CMakeDriver implements vscode.Disposable {
       subs.set(full, repl);
     }
 
+    const env2_re = /\$\{env\.(.+?)\}/g;
+    while ((mat = env2_re.exec(instr))) {
+      const full = mat[0];
+      const varname = mat[1];
+      const repl = process.env[varname.toLocaleLowerCase()] || '';
```

Comment:
Is `varname.toLocaleLowerCase()` correct? On Windows env vars are case insensitive, but Unix respects case.

Reply:
I can add platform detection, but I really just copied the already existing one and modified the regex.

Reply:
Then _I_ probably did a wrong. May be prudent to move reading of environment variables to `util.ts` so that we can switch on case-sensitivity.

Reply:
I will add a commit for that after I have gotten some sleep.
Have been working on hypervisor stuff for a couple hours and my mind isn't working anymore.

Reply:
Can you move out the case-normalization to `util.ts`?

There's an itching in the back of my mind that lower-casing environment variables on Linux may be to blame for one or more bug reports I've seen in the past.